server/controllers/CommentsController.js

```diff
@@ -167,8 +167,8 @@ class CommentsControllers {
       }).then((comments) => {
         const { count } = comments;
         const pageCount = Math.ceil(count / limit);
-        if (count < 1) {
-          return res.status(404).json({
+        if (comments.rows.length < 1) {
```

Comment:
Take ou this check. A user can make a request to page 50 which will return an empty result but that does not imply there are no comments on the article.